bundler/spec/runtime/inline_spec.rb

```diff
@@ -344,15 +344,25 @@ def confirm(msg, newline = nil)
 
   it "skips platform warnings" do
     bundle "config set --local force_ruby_platform true"
+    script <<-RUBY
+      gemfile(true) do
+        source "#{file_uri_for(gem_repo1)}"
+        gem "rack", platform: :jruby
+      end
+    RUBY
 
+    expect(err_without_deprecations).to be_empty
+  end
+
+  it "shows platform warnings" do
     script <<-RUBY
       gemfile(true) do
         source "#{file_uri_for(gem_repo1)}"
         gem "rack", platform: :jruby
       end
     RUBY
 
-    expect(err).to be_empty
+    expect(err).to eq("[DEPRECATED] 'install' argument. The installation will happen for missing gems. It is not necessary pass a 'true' value.")
```

Comment:
For consistency with other deprecation specs, use `expect(deprecations).to include` and remove the `[DEPRECATED]` part.

Reply:
Ok.